README.md

```diff
@@ -11,21 +11,52 @@ The Extensions API lets you do more without leaving Tableau. Build Tableau exten
+### Install Extensions API SDK Components and Start Server
 
-### Typescript Development
-Samples written in Typescript are located in the Samples-Typescript folder. To support local typescript development, `npm run dev` command starts up the http server and actively listens for changes to the .ts files located in the Sample-Typescript folder.
+1. Open a command prompt window to the location where you cloned this repo.
+
+2. Install the Extensions API SDK components.
+
+    **npm install**
+
+3. Build the TypeScript samples and install the Extensions API types library.
+
+   **npm run build**
+
+4. Start the local Dashboard Extension server.
 
- >**Note** The web server just serves the extension samples and tutorial, which have URLs similar to the following: `http://localhost:8765/Samples/DataSources/datasources.html` or `http://localhost:8765/Samples-Typescript/DataSources/datasources.html`
+   **npm start**
+
+5. Launch Tableau and try the sample extensions in a dashboard. The samples are located in the `Samples` folder.
+
+ >**Note** The local web server you start just serves to host the extension samples and extensions used in the tutorial, which have URLs similar to the following: `http://localhost:8765/Samples/DataSources/datasources.html` or `http://localhost:8765/Samples-Typescript/DataSources/datasources.html`
  >   This local web server is not intended to serve the Extensions API Help pages. 
  >   View the Help on GitHub at [https://tableau.github.io/extensions-api](https://tableau.github.io/extensions-api).
 
+### Typescript Development
+Samples written in Typescript are located in the `Samples-Typescript` folder.
+If you want to use TypeScript to write your extensions, you can run a script that starts up the HTTP server and actively listens for changes to the `.ts` files located in the `Sample-Typescript` folder. You can then add your extension to the folder and use the script to transpile your extension to JavaScript.
+
+* To start the the HTTP server and listen for changes to the `.ts` files.
+
+    **npm run dev**
+
+For more information, see [Use TypeScript with the Extensions API](https://tableau.github.io/extensions-api/docs/trex_typescript.html).
+
+
+### Sandboxed Extension Development Environment
+
+Tableau is introducing development support for Sandboxed Extensions with Tableau 2019.3. Sandboxed Extensions run in a virtual sandbox and ensure the extension can’t make network calls outside of the hosting Tableau Server. The Extensions API SDK provides a local development environment that replicates the Tableau Hosting Cloud Service for Sandboxed Extensions. You can test your Sandboxed extensions locally with the same sandbox policies.
+
+1. Start the local Sandboxed Extension web server.
+
+    **npm run start-sandbox**
+
+2. Launch Tableau (Tableau 2019.3 and later) and try the sample Sandboxed Extension in a dashboard. The sample is located in the `Samples\UINamespace-sandboxed` folder. 
```

Comment:
maybe something about the trex & sample code is in the x folder

Reply:
Sounds good. I'm assuming that they will know how to add the extension at this point. 

You can find the `.trex` file and sample code in the `Samples\UINamespace-sandboxed` folder.